presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -25,6 +25,29 @@ data warehouse. Hive is a combination of three components:
 Presto only uses the first two components: the data and the metadata.
 It does not use HiveQL or any part of Hive's execution environment.
 
+Requirements
+------------
+
+The Hive connector requires a Hive metastore service (HMS). The following
+versions are supported:
+
+* Apache Hive 1.x
+* Apache Hive 2.x
+
+The following Hadoop distributions, and the included HMS, are supported.
```

Comment:
This is incorrect, since Hadoop is just base libraries, MapReduce, and HDFS. The Hive metastore comes from the Hive project.